src/main/java/org/jabref/logic/integrity/FieldCheckers.java

```diff
@@ -53,6 +53,11 @@ private static Multimap<Field, ValueChecker> getAllMap(BibDatabaseContext databa
             fieldCheckers.put(StandardField.URLDATE, new DateChecker());
             fieldCheckers.put(StandardField.EVENTDATE, new DateChecker());
             fieldCheckers.put(StandardField.ORIGDATE, new DateChecker());
+        } else {
+            // field checkers for the bibtex mode only
+            for (Field field : FieldFactory.getStandardFieldsWithCitationKey()) {
```

Comment:
This "feels" wrong. What do other checkers do? I think, they check all text fields (and not just the standard fields)